feature/onboarding/src/main/java/com/teamwable/onboarding/profile/regex/NicknameValidationUseCase.kt

```diff
@@ -0,0 +1,23 @@
+package com.teamwable.onboarding.profile.regex
+
+import com.teamwable.designsystem.type.NicknameType
+import javax.inject.Inject
+
+class NicknameValidationUseCase @Inject constructor() {
```

Comment:
```suggestion
class NicknameValidationUseCase {
```
P2 : 주입할거 없으면 지워도 될듯합니다!

Reply:
엇 뷰모델의 파라미터로 넣으려면 이게 필요해요!

Reply:
홀리몰리 제가 착각했네요 good